hawkbit-artifact-repository-filesystem/src/main/java/org/eclipse/hawkbit/artifact/repository/ArtifactFilesystem.java

```diff
@@ -47,4 +52,19 @@ public InputStream getFileInputStream() {
             throw new ArtifactFileNotFoundException(e);
         }
     }
+
+    @Override
+    // suppress warning, this InputStream needs to be closed by the caller, this
+    // cannot be closed in this method
+    @SuppressWarnings("squid:S2095")
+    public InputStream getFileInputStream(long start, long end) {
+        try {
+            var f = new RandomAccessFile(file, "r");
+            var ch = f.getChannel();
```

Comment:
I'd rather not add an additional dependency on com.google.common
wouldn't 
```
f.seek(start);
return f;
```
do the same?